circuitpy/lib/farm_ng/utils/packet.py

```diff
@@ -98,7 +102,6 @@ def encode(self):
 
     def decode(self, data):
         """Decodes CAN message data and populates the values of the class."""
-
         (xi, yi, self.buttons) = unpack(self.format, data)
         self.x = xi / 32767
```

Comment:
Prefer magic numbers at least commented if not using a define. MAX16BIT_NUM = 32767